src/styles/index.css

```diff
@@ -0,0 +1,275 @@
+.body {
+  color: #060b35;
+  margin: 0;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  list-style: none;
+  position: relative;
+  margin-left: 20px;
```

Comment:
Consider using relative units (like percentages or ems) for margin to improve responsiveness.